frontend/src/components/ChainInfo.vue

```diff
@@ -35,15 +35,13 @@
 							<q-tooltip content-class="bg-positive">Deposit some BNB for swaps fees</q-tooltip>
 						</q-btn>
 					</div>
-					<!--
 					<div class="col-3">
 						<q-btn @click="reloadBalances" color="primary" stack>
 							<q-icon name="img:ban-refresh.svg" size="3em" color="secondary" />
 							<div class="text-button">Refresh Balances</div>
 							<q-tooltip content-class="bg-positive">Refresh balances</q-tooltip>
 						</q-btn>
 					</div>
-					-->
```

Comment:
The idea is actually to get rid of this button has balances should be updated by server-side events.
This was more confusing than anything else.